plugins/kubernetes/app/views/admin/kubernetes/clusters/_form.html.erb

```diff
@@ -35,6 +35,14 @@
     </div>
 
     <div class="form-group">
+      <%= form.label :ip_prefix, "IP prefix", class: "col-lg-2 control-label" %>
+      <div class="col-lg-4">
+        <%= form.text_field :ip_prefix, class: "form-control" %>
+      </div>
+      <%= additional_info("will replace service clusterIP") %>
```

Comment:
Could you describe this a little better? Explain that it's the first 1 to 3 sections of an IPv4 address, and it will replace the ClusterIP if specified.